### DISCUSSION TITLE
Object.hash not found in flutter stable (2.2.3)

### DISCUSSION THREAD

**Discussion by jeduden** [rrousselGit/riverpod#726] [General]
Hi ! 
I was just trying the head of master with Flutter stable (2.2.3) with riverpod's head of master, since I wanted to use the latest changes. I got compile time errors like:

```
[...]/packages/riverpod/lib/src/common.dart:173:30: Error: Method not found: 'Object.hash'.
  int get hashCode => Object.hash(runtimeType, value);  
```

when I changed to flutter's beta channel it compiled just fine. is that a known behaviour and intended ?

Jan-Eric

**Comment by tbm98** [comment]
if you are using riverpod dev version. It is target to run on flutter dev channel now

**Reply by TimWhiting** [reply]
@rrousselGit do you want to set the minimum flutter sdk for this?

**Reply by rrousselGit** [reply]
Ah, I forgot to change it. Will do

**Reply by jeduden** [reply]
thanks !

**Comment by rrousselGit** [comment]
As mentioned, you will need a Dart SDK >= 2.14.0
